### ISSUE TITLE
Cannot view a variable in Variable explorer

### ISSUE THREAD

**Issue by szoto5** [spyder-ide/spyder#8887]
### What steps will reproduce the problem?

1. Loaded pandas as pd
2. Loaded a csv file. Once loaded it is of type DataFrame and I can open it from Variable Explorer 
dataset = pd.read_csv('filename.csv')
3. Wanted to take the last column of the above that included just zeros and ones:
y = dataset.iloc[:, 13].values
4. It has been created in variable explorer as type=int64 but after clicking and wanting to see a preview I got this error.

### Traceback
```python-traceback
  File "C:\ProgramData\Anaconda3\lib\site-packages\spyder\widgets\variableexplorer\collectionseditor.py", line 476, in createEditor
    if not editor.setup_and_check(value, title=key, readonly=readonly):
  File "C:\ProgramData\Anaconda3\lib\site-packages\spyder\widgets\variableexplorer\arrayeditor.py", line 620, in setup_and_check
    self.data.flags.writeable = True
ValueError: cannot set WRITEABLE flag to True of this array
```

## Versions

* Spyder version: 3.3.1 
* Python version: 3.6.7
* Qt version: 5.9.6
* PyQt5 version: 5.9.2
* Operating System: Windows 7

### Dependencies

```
pyflakes >=0.6.0  :  2.1.1 (OK)
pycodestyle >=2.3 :  2.5.0 (OK)
pygments >=2.0    :  2.3.1 (OK)
sphinx >=0.6.6    :  1.8.4 (OK)
rope >=0.9.4      :  0.10.7 (OK)
jedi >=0.9.0      :  0.13.3 (OK)
nbconvert >=4.0   :  5.4.1 (OK)
pandas >=0.13.1   :  0.24.1 (OK)
numpy >=1.7       :  1.16.2 (OK)
sympy >=0.7.3     :  1.3 (OK)
cython >=0.21     :  0.29.6 (OK)
qtconsole >=4.2.0 :  4.4.3 (OK)
IPython >=4.0     :  7.3.0 (OK)
matplotlib >=2.0.0:  3.0.3 (OK)
pylint >=0.25     :  2.2.2 (OK)
```

**Comment by ccordoba12** [comment]
This is fixed in our latest version (3.3.3). Please update by opening the Anaconda Prompt and running there

    conda update spyder

**Comment by szoto5** [comment]
It works fine. Thanks a lot!
